### PR TITLE
Update Android migration page SDK version

### PR DESCRIPTION
**Pull request by marandaneto**

### CODE REVIEW

File: src/platforms/android/migration.mdx [hunk 1]
```diff
@@ -5,7 +5,7 @@ redirect_from:
   - /platforms/android/migrate/
 ---
 
-## Migrating from `sentry-android 2.x` to `sentry-android 3.x`
+## Migrating from `sentry-android 2.x` to `sentry-android 4.x`
```
Comment (philipphofmann) [inline src/platforms/android/migration.mdx:8]: `m`: I would rather keep this and state that there is no migration needed between 3.x to 4.x.

Comment (bruno-garcia) [reply src/platforms/android/migration.mdx:8]: Agreed. Could even say that: The performance API was added but API-wise no breaking changes exist.

Also, I find it hard to believe there are 0 breaking changes :D

Comment (bruno-garcia) [reply src/platforms/android/migration.mdx:8]: Oh it's Android specific. I believe it now

Comment (marandaneto) [reply src/platforms/android/migration.mdx:8]: will just close it then

**Pull request closed without merging**